src/mod/gcewing/sg/features/cc/CCSGPeripheral.java

```diff
@@ -36,7 +36,7 @@
         
         new SGMethod("energyToDial", 1) {
             Object[] call(SGInterfaceTE te, Object[] args) {
-                return new Object[] {te.ciEnergyToDial((String)args[0])};
+                return new Object[] {te.ciEnergyToDial((String)args[0], false)};
```

Comment:
@Dockter I am hedging that ComputerCraft doesn't do it on TE tick. It very well might. If so, flip these.